jbpm-bom/pom.xml

```diff
@@ -815,6 +815,12 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-wb-case-mgmt-showcase</artifactId>
+        <version>${version.org.kie}</version>
+        <type>war</type>
+      </dependency>
```

Comment:
@cristianonicolai thanks for the details. What exactly does it mean? Should we publishing it on the jbpm.org pages? Or will it just be an link to the Nexus? We will do additional testing for that war?